### ISSUE TITLE
Error: Processing Response error

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-launch-dapp#708]

**Comment by dkent600** [comment]
Yeah, I discovered that today too, but haven't have time to work it

**Comment by dkent600** [comment]
This was do to the cap Infura imposes on the number of blocks one can query at a time (100,000).  Fixed by switching from Infura to https://arb1.arbitrum.io/rpc
